### PR TITLE
Project popup

### PR DESCRIPTION
**Pull request by Omar-Muhamad**
- [ ] Add data dynamically to the DOM using JavaScript to make the project pop up.
- [ ] Add styling to mobile version.
- [ ] Add styling to desktop version.
- [ ] Add all linters files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -144,7 +144,7 @@ <h2 class="projectName">Facebook 360</h2>
             <li class="lang">css</li>
             <li class="lang">javaScript</li>
           </ul>
-          <button type="button">See Project</button>
+          <button class="pop-btn" id="2" type="button">See Project</button>
```
Comment (rotimiazeez) [inline index.html:147]: - [ ] Please note that the should popup follows the design for both desktop and mobile version from the chosen template in Figma (e.g., font, colours, images, tests, margins).
-Your Design
screenshot (6)
- Figma Design
screenshot (7)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team 👋, 

### Good job so far!💯

### To highlight 👏
- Good UI 👌
- Good README 👍
- Descriptive PR message 😉
- No Linter error ✔
#### There are some issues that you still need to work on to go to the next project but you are almost there!
#### Just a few steps and you are unstoppable😎

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Omar-Muhamad** [comment]
Fix reviewer requirement and made popup card scrollable but hide the scroll bar to stick to the design.

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi team 👋🏽  

Great job implementing the changes requested by the previous reviewer 👏🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**